Makefile

```diff
@@ -20,20 +20,36 @@ test: build
 	CUDA_VISIBLE_DEVICES=$(CUDA_VISIBLE_DEVICES) bash ./blueoil_test.sh
 
 .PHONY: test-lmnet
-test-lmnet: build
+test-lmnet: test-lmnet-pep8 test-lmnet-main test-lmnet-dataset
+
+.PHONY: test-lmnet-pep8
+test-lmnet-pep8: build
+	# Check lmnet pep8
+	docker run --rm $(IMAGE_NAME):$(BUILD_VERSION) /bin/bash -c "cd lmnet; tox -e flake8"
+
+.PHONY: test-lmnet-main
+test-lmnet-main: build
 	# Run lmnet test with Python3.6 (only available on Jenkins)
-	docker run $(DOCKER_OPT) -v /storage/dataset:/storage/dataset -v lmnet_saved:/storage/lmnet/saved -e CUDA_VISIBLE_DEVICES=$(CUDA_VISIBLE_DEVICES) -e DATA_DIR=/storage/dataset --rm $(IMAGE_NAME):$(BUILD_VERSION) /bin/bash -c "cd lmnet; tox -e py36"
+	docker run $(DOCKER_OPT) -v /storage/dataset:/storage/dataset -v lmnet_saved:/storage/lmnet/saved -e CUDA_VISIBLE_DEVICES=$(CUDA_VISIBLE_DEVICES) -e DATA_DIR=/storage/dataset --rm $(IMAGE_NAME):$(BUILD_VERSION) /bin/bash -c "cd lmnet; tox -e py36-pytest"
+
+.PHONY: test-lmnet-dataset
+test-lmnet-dataset: build
```

Comment:
The `test-lmnet-dataset` name and description `Run lmnet dataset class test and check datasets` makes developer confuse because `check_dataset_storage.py` mainly check dataset storage but not mainly test datasets. 


This is from `check_dataset_storage.py` file doc.
> This is the checker of dataset storage,
check the each datasets is stored corect on storage, and the number of images in each datasets.
And also test that each dataset class don't get error.
Normally developer don't needs to path all tests
because they only forcus some of dataset, not needs the all of datasets.

https://github.com/blue-oil/blueoil/blob/master/lmnet/tests/check_dataset_storage.py#L19-L23




I recommend to use
`test-lmnet-check-dataset-storage`
 and
`Check datasets storage with Python3.6 (only available on Jenkins)`

Reply:
@ruimashita Thanks! I changed them following your suggestion. Plz check :+1: